src/utilities/cache-manager.js

```diff
@@ -25,17 +25,20 @@ export default class CacheManager {
   }
 
   createCache(name) {
-    internalCaches[name] = LRU({
-      max: parseInt(process.env.CACHE_MAX_ITEM_COUNT, 10) || 100,
-      maxAge: parseInt(process.env.CACHE_MAX_AGE, 10) || 1
+    const count = parseInt(process.env.CACHE_MAX_ITEM_COUNT, 10) || 100
+    const maxAge = parseInt(process.env.CACHE_MAX_AGE, 10) || 1
+    internalCaches[name] = NCM.caching({
+      store: 'memory',
```

Comment:
This is where I'll put in some of the caching configuration logic to allow plugins within a project.